packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -180,7 +189,7 @@ contract DeployImplementationsOutput {
         require(false, "DeployImplementationsOutput: not implemented");
     }
 
-    function checkOutput() public {
+    function checkOutput(DeployImplementationsInput) public {
```

Comment:
I noticed `DeployImplementationsInput` isn't being used in this function. Should it be?

Reply:
Nope, it's just there because so all `checkOutput` methods have a consistent function signature of `checkOutput(inputContract)`, even if unused. Similar to how `deploy*` methods always take the input and output contract addresses as inputs even when not used—this consistency mitigates the risk of breaking changes in the future